src/main/java/com/autosale/shop/security/JwtAuthenticationFilter.java

```diff
@@ -0,0 +1,47 @@
+package com.autosale.shop.security;
+
+import com.autosale.shop.service.JwtTokenService;
+import jakarta.servlet.FilterChain;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpHeaders;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Component;
+import org.springframework.web.filter.OncePerRequestFilter;
+
+import java.io.IOException;
+
+@Component
+@RequiredArgsConstructor
+public class JwtAuthenticationFilter extends OncePerRequestFilter {
+
+    private final JwtTokenService jwtTokenService;
+
+    public static final String JwtTokenPrefix = "Bearer ";
```

Comment:
1. follow the naming convention for constants.
2. Constants should be upper than private final fields
3.  Why public? Do you use it out of this class?